src/components/AutomationDetails/index.jsx

```diff
@@ -1,126 +1,181 @@
     return (
-      <div className="table-header-details">
-        <table className="details-table">
-          <thead>
-            <tr>
-              {modalType === 'email' ? <th>Sent to</th> : <th>Channel</th>}
-              {modalType === 'email' ? <th>Subject</th> : <th>Auto Type</th>}
-              <th>Status</th>
-            </tr>
-          </thead>
-        </table>
-        <div className="automation-table">
-          {contents.map((content) => {
-            const { emailTo } = content;
-            return (
-              <div key={modalType === 'email' ? emailTo : uuid.v4()} className="table-body-details">
-                <table className="details-table">
-                  <tbody>
-                    <tr>
-                      {this.renderTableData(content, modalType)}
-                    </tr>
-                  </tbody>
-                </table>
-              </div>
-            );
-          })}
+      <div className="channel-tabs-group">
+        <div className="channels-tab-background">
+          <button type="button" className={`automations ${modalType === 'slack' ? 'active' : 'deactivated'}`} id="slack" onClick={this.setCurrentAutomationType}>Slack</button>
+          <button type="button" className={`automations ${modalType === 'email' ? 'active' : 'deactivated'}`} id="email" onClick={this.setCurrentAutomationType}>Email</button>
+          <button type="button" className={`automations ${modalType === 'freckle' ? 'active' : 'deactivated'}`} id="freckle" onClick={this.setCurrentAutomationType}>Freckle</button>
         </div>
       </div>
     );
   }
 
-  renderTableData = (content, modalType) => {
-    const {
-      channelName, type, emailTo, subject, status,
-    } = content;
-    let automationMedia = channelName;
-    let automationTitle = type;
-    if (modalType === 'email') {
-      automationMedia = emailTo;
-      automationTitle = subject;
-    }
+  renderTitles(modalType) {
+    return (
+      <div>
+        {modalType === 'slack' && this.renderSlackTitles()}
+        {modalType === 'email' && this.renderEmailTitles()}
+        {modalType === 'freckle' && this.renderFreckleTitles()}
+      </div>
+    );
+  }
+
+  renderSlackTitles = () => (
+    <div className="slack-table">
+      <div className="content-title">Channel</div>
+      <div className="content-title">Action</div>
+      <div className="content-title">Status</div>
+    </div>
+  );
+
+  renderEmailTitles = () => (
+    <div className="slack-table">
+      <div className="content-title">Recipient</div>
+      <div className="content-title">Subject</div>
+      <div className="content-title">Status</div>
+    </div>
+  );
+
+  renderFreckleTitles = () => (
+    <div className="slack-table">
+      <div className="content-title">Project Tag</div>
+      <div className="content-title">Action</div>
+      <div className="content-title">Status</div>
+    </div>
+  );
+
+  renderDetails(modalType, modalContent) {
```

Comment:
I think you should employ arrow function technique here `renderDetails = (modalType, modalContent) => ()`, since it automatically bind `this` keyword. What do you think?